linode/manager
# refactor: [M3-6275] - MUI v5 Migration - SRC > Features > Managed

## Description 📝
- MUI style migration for Managed feature

## Major Changes 🔄
### Managed/Contacts
- [x] Contacts.tsx
- [x] ContactsActionMenu.tsx
- [x] ContactsDrawer.tsx
- [x] ContactsRow.tsx
- [x] ContactsTableContent.tsx

### Managed/Credentials
- [x] AddCredentialsDrawer.tsx
- [x] CredentialActionMenu.tsx
- [x] CredentialList.tsx
- [x] CredentialRow.tsx
- [x] CredetialTableContent.tsx
- [x] UpdateCredetialDrawer.tsx

### Managed/ManagedDashboardCard
- [x] DashboardCard.tsx
- [x] ManagedChartPanel.tsx
- [x] ManagedDashboardCard.tsx
- [x] MonitorStatus.tsx
- [x] MonitorTickets.tsx

### Managed/Monitors
- [x] HistoryDrawer.tsx
- [x] IssueCalendar.tsx
- [x] IssueDay.tsx
- [x] MonitorActionMenu.tsx
- [x] MonitorRow.tsx
- [x] MonitorTable.tsx
- [x] MonitorTableContent.tsx

### Managed/SSH
- [x] EditSSHAccessDrawer.tsx
- [x] LinodePublicKey.tsx
- [x] SSHAccess.tsx
- [x] SSHAccessActionMenu.tsx
- [x] SSHAccessRow.tsx
- [x] SSHAccessTable.tsx
- [x] SSHAccessTableContent.tsx

### Managed/
- [x] ManagedLanding.tsx
- [x] MonitorDrawing.tsx
- [x] SupportWidget.tsx

## Preview 📷
| Before      | After |
| ----------- | ----------- |
| ![Screenshot 2023-07-31 at 12 11 54 PM](https://github.com/linode/manager/assets/139489745/260ad864-b9a0-458b-8774-b43c9bff56b2) | ![Screenshot 2023-07-31 at 12 20 19 PM](https://github.com/linode/manager/assets/139489745/fed6bd88-c179-4319-b24b-4403e72fe2fa) |
| ![Screenshot 2023-07-31 at 12 12 03 PM](https://github.com/linode/manager/assets/139489745/7387d6d0-60e7-4429-b4e1-006fe9d10d50) | ![Screenshot 2023-07-31 at 12 20 27 PM](https://github.com/linode/manager/assets/139489745/0a32f0ec-7ae8-4561-8f08-a386fbebf94c) | 
| ![Screenshot 2023-07-31 at 12 12 42 PM](https://github.com/linode/manager/assets/139489745/e602b389-e3b6-4417-8031-0f06f94f3702) | ![Screenshot 2023-07-31 at 12 20 36 PM](https://github.com/linode/manager/assets/139489745/cb24b964-844b-47c4-a768-7f740aac89e9) | 
| ![Screenshot 2023-07-31 at 12 12 11 PM](https://github.com/linode/manager/assets/139489745/8ce37d88-7e2d-4a84-aaac-5765f4bd0005) | ![Screenshot 2023-07-31 at 12 20 45 PM](https://github.com/linode/manager/assets/139489745/46c44c09-f56f-4aba-b0cb-cc6eb685a9fd) | 
| ![Screenshot 2023-07-31 at 12 12 24 PM](https://github.com/linode/manager/assets/139489745/f6b755b2-cf7d-4274-b37a-48f80b9f194a) | ![Screenshot 2023-07-31 at 12 20 54 PM](https://github.com/linode/manager/assets/139489745/e854d767-b79a-49fc-ae88-a5f30572521c) | 
| ![Screenshot 2023-07-31 at 12 12 55 PM](https://github.com/linode/manager/assets/139489745/9f77a465-c74c-49dd-a1f1-186721497d2e) | ![Screenshot 2023-07-31 at 12 21 02 PM](https://github.com/linode/manager/assets/139489745/fda4665f-96ac-4a1e-b3a6-210a787ec027) | 
| ![Screenshot 2023-07-31 at 12 21 10 PM](https://github.com/linode/manager/assets/139489745/dc8328b3-7a6e-4903-9b50-de479964a7c6) | ![Screenshot 2023-07-31 at 12 21 10 PM](https://github.com/linode/manager/assets/139489745/16252135-26f1-4bb5-b049-eb7e545d626f) |
| ![Screenshot 2023-07-31 at 12 21 16 PM](https://github.com/linode/manager/assets/139489745/03df3a62-d714-4aaf-a1b2-c0793ed7210f) | ![Screenshot 2023-07-31 at 12 21 16 PM](https://github.com/linode/manager/assets/139489745/bc907443-2674-42d1-a159-1ae91feb1f66) |

## How to test 
If you don't have Managed configured:
1. Navigate to Account Settings
2. Enable Linode Managed
3. There is a new Managed tab that pops up, navigate to the new Managed tab
4. Make sure the displays look consistent with the develop branch
